_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -0,0 +1,320 @@
+
+Each corresponding example in the [examples/](https://github.com/forseti-security/terraform-google-forseti/tree/master/examples/) folder on GitHub also contains additional information on each deployment scenario.
+
+Wherever possible, the examples utilize [modules](https://registry.terraform.io/modules/terraform-google-modules) developed and curated by the [Cloud Foundation Toolkit](https://cloud.google.com/foundation-toolkit/) team.  These modules implement opinionated best practices for deploying GCP components.  For example, the [kubernetes-engine](https://registry.terraform.io/modules/terraform-google-modules/kubernetes-engine/google/4.0.0) module applies practices found in the [GKE hardening](https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster).
+
+### Deploy Forseti and Forseti on GKE end-to-end
+
+Create a file named *main.tf* in an empty directory and add the following content.  Add the appropiate values for each of the input variables (e.g. domain, gsuite_admin_email).
+
+```bash
+module "forseti-on-gke-end-to-end" {
+    source                  = "terraform-google-modules/forseti/google//examples/on_gke_end_to_end"
+    credentials_path        = ""
+    domain                  = ""
+    gsuite_admin_email      = ""
+    org_id                  = ""
+    project_id              = ""
+    region                  = ""
+    zones                   = [""]
+    network_description     = "GKE Network"
+    auto_create_subnetworks = false
+    gke_node_ip_range       = "10.1.0.0/20"
+}
+```
+
+Initialize the Terraform module.
+
+```bash
+terraform init
+```
+
+Apply the Terraform module.
+
+```bash
+terraform apply
+```
+
+**Note:** The `terraform apply` may fail to complete.  In this case, please re-run `terraform apply`.
+
+### Deploy Forseti in a New GKE Cluster
+
+Create a file named *main.tf* in an empty directory and add the following content.  Add the appropiate values for each of the input variables (e.g. domain, gsuite_admin_email).
+
+```bash
+module "forseti-on-gke-new-gke-cluster" {
+    source                           = "terraform-google-modules/forseti/google//examples/on_gke_new_gke_cluster"
+    credentials_path                 = ""
+	# SERVICE_ACCOUNT_NAME@PROJECT_ID.iam.gserviceaccount.com
+    forseti_client_service_account   = "" 
+    forseti_client_vm_ip             = ""
+    forseti_cloudsql_connection_name = ""
+	# SERVICE_ACCOUNT_NAME@PROJECT_ID.iam.gserviceaccount.com
+    forseti_server_service_account   = ""
+	# BUCKET_ID (without gs)
+    forseti_server_storage_bucket    = ""
+    project_id                       = ""
+    region                           = ""
+    suffix                           = ""
```

Comment:
Would be good to mention where you can look for the suffix (e.g. whatever is appended after `forseti-server-xxx`